README.md

```diff
@@ -79,3 +79,50 @@ client, _ := planetscale.NewClient(
 	planetscale.WithAccessToken(token),
 )
 ```
+
+## Connecting to a PlanetScale Database
+
+The `planetscale-go` packge provides a helper method to simplify connecting to a PlanetScale database. Here is an example you can use (_Please make sure to handle errors in your production application._):
+
+
+
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/planetscale/planetscale-go/planetscale"
+	"github.com/planetscale/planetscale-go/planetscale/dbutil"
+)
+
+func main() {
+	token := os.Getenv("PLANETSCALE_TOKEN")
```

Comment:
It might be worth mentioning that this is a token brought from `pscale auth login` and not from `pscale service-token create` (unless that's something that's gonna be allowed in the future.)

Reply:
You're right. I think we should update our docs to show also an example with service tokens, because we have already support for service tokens:

```go
// WithServiceToken configures a client with the given PlanetScale Service Token
func WithServiceToken(name, token string) ClientOption {
	return func(c *Client) error {
		if token == "" || name == "" {
			return errors.New("missing token name and string")
		}

		transport := serviceTokenTransport{
			rt:        c.client.Transport,
			token:     token,
			tokenName: name,
		}

		c.client.Transport = &transport
		return nil
	}
}
```

However, we don't show them yet in our Readme. We should probably update them all.

Reply:
Ah, that looks great. Thanks